tests/smoketests/test_coalescent.py

```diff
@@ -0,0 +1,103 @@
+import dendropy
+from . import pytestmark
+from dendropy.datamodel.treemodel import Tree, Node
+
+from dendropy.model.coalescent import (
+    discrete_time_to_coalescence,
+    time_to_coalescence,
+    expected_tmrca,
+    coalesce_nodes,
+    node_waiting_time_pairs,
+    extract_coalescent_frames,
+    log_probability_of_coalescent_frames,
+    log_probability_of_coalescent_tree,
+    contained_coalescent_tree,
+    pure_kingman_tree,
+    pure_kingman_tree_shape,
+    mean_kingman_tree,
+    constrained_kingman_tree,
+)
+
+
+def test_discrete_time_to_coalescence():
+    # discrete_time_to_coalescence(2) # if no second parameter, errors on 72
+    # discrete_time_to_coalescence(1, 1) # first parameter must be >1
+    discrete_time_to_coalescence(2, 1)
+    assert type(discrete_time_to_coalescence(2, 1, 1)) == int
+
+def test_time_to_coalesence():
+    # time_to_coalescence(1) # also requires >1
```

Comment:
let's just change this to a comment instead of having the commented out code, it's always a bit confusing to come back to commented code because you're never quite sure why it was commented out